### PR TITLE
Php Agent Metadata

### PR DESCRIPTION
**Pull request by bduranleau-nr**
Implements Agent Metadata for all currently supported releases of the PHP Agent .

### CODE REVIEW

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx [hunk 1]
```diff
@@ -3,6 +3,9 @@ subject: PHP agent
 releaseDate: '2022-03-02'
 version: 9.19.0.309
 downloadLink: 'https://download.newrelic.com/php_agent/archive/9.19.0.309'
+features: ['Add support for 64-but PHP 8.1']
```
Comment (mfulb) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx:6]:
```suggestion
features: ['Add support for 64-bit PHP 8.1']
```

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-1010313.mdx [hunk 2]
```diff
@@ -3,6 +3,9 @@ subject: PHP agent
 releaseDate: '2022-09-19'
 version: 10.1.0.313
 downloadLink: 'https://download.newrelic.com/php_agent/archive/10.1.0.313'
+features: ['Add APM logs in context for Monolog 2 and 3', 'Enhanced redis/PhpRedis support']
```
Comment (bduranleau-nr) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-1010313.mdx:6]:
```suggestion
features: ['Add APM logs in context for Monolog 2 and 3', 'Enhanced redis and PhpRedis support']
```

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9101263.mdx [hunk 3]
```diff
@@ -3,6 +3,9 @@ subject: PHP agent
 releaseDate: '2020-05-05'
 version: 9.10.1.263
 downloadLink: 'https://download.newrelic.com/php_agent/archive/9.10.1.263'
+features: []
+bugs: ['Fix Laravel queue w/distributed tracing segfault']
```
Comment (bduranleau-nr) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9101263.mdx:7]:
```suggestion
bugs: ['Fix Laravel queue with distributed tracing segfault']
```

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9120268.mdx [hunk 4]
```diff
@@ -5,6 +5,9 @@ version: 9.12.0.268
 downloadLink: 'https://download.newrelic.com/php_agent/archive/9.12.0.268/'
 redirects:
   - /docs/release-notes/agent-release-notes/php-release-notes/php-agent-9120
+features: ['Add span attributes', 'Add Infinite Tracing support']
+bugs: ['Fix agent/daemon IP address resolution for separate hosts']
```
Comment (bduranleau-nr) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9120268.mdx:9]:
```suggestion
bugs: ['Fix agent to daemon IP address resolution for separate hosts']
```

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9181303.mdx [hunk 5]
```diff
@@ -3,6 +3,9 @@ subject: PHP agent
 releaseDate: '2021-08-23'
 version: 9.18.1.303
 downloadLink: 'https://download.newrelic.com/php_agent/archive/9.18.1.303'
+features: ['Add Docker development environment support', 'Add support for Route caching in Laravel 7.x', 'Add support for Redis::mget and Redis::mset functions']
```
Comment (bduranleau-nr) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9181303.mdx:6]:
```suggestion
features: ['Add Docker development environment support', 'Add support for Route caching in Laravel 7.x', 'Add support for "Redis::mget" and "Redis::mset" functions']
```

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx [hunk 6]
```diff
@@ -3,6 +3,9 @@ subject: PHP agent
 releaseDate: '2022-10-03'
 version: 10.2.0.314
 downloadLink: 'https://download.newrelic.com/php_agent/archive/10.2.0.314'
+features: ['Alert when number of connected applications falls below 25 or exceeds 200']
+bugs: ['Fix SIGSEGV in Drupal 7 instrumentation', 'Fix PHPinfo output for newrelic.application_logging*.enabled', 'Fix JSON string formatting for application log messages', 'Reduce severity of log message when the agent does not detect environmental variables']
```
Comment (bduranleau-nr) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx:7]:
```suggestion
bugs: ['Fix SIGSEGV in Drupal 7 instrumentation', 'Fix PHPinfo output for "newrelic.application_logging*.enabled"', 'Fix JSON string formatting for application log messages', 'Reduce severity of log message when the agent does not detect environmental variables']
```

**Comment by ally-sassman** [comment]
@bduranleau-nr Do we need another review from @mfulb , or is this ready to merge?

**Comment by bduranleau-nr** [comment]
Hi @ally-sassman - The reviews are all set, was waiting for the Gatsby Build Service to finish so I could preview the metadata, but it seems to have been stuck in a building state for ~3 days. Not sure if there's an issue with the build service or something in the PR itself.

**Comment by ally-sassman** [comment]
@bduranleau-nr Oh sure, let me try retriggering the build by closing/reopening the PR. Often times that does the trick. If not, I'll slack #doc-eng-help  to help out.

**Pull request merged**